Webex Connect Flows/v3.0/Sample/Bot Flows/README.md

```diff
@@ -0,0 +1,39 @@
+# Instructions for using bot flow templates
+
+## Q&A bot flow for Livechat channel
+Pre-requisites
+1.	A Webex Connect tenant with bot builder app, WxCC nodes, and Livechat channel enabled
+2.	A Livechat app in connect
+      * A website configuration in Webex Engage for the Livechat asset – this is required in the Create Conversation node.
+      * Pre-chat form template in Webex Connect if you want to use a pre-chat form. The nodes specific to gathering pre-chat form can be deleted from the flow if you choose       to not use a pre-chat form.
+3. Q&A bot in the bot builder
+    * A Q&A bot with desired articles should be created in the bot builder app. This will create an imibot integration in connect, which can then be selected from a       dropdown in the Q&A bot node.
+    * The Q&A bot should be trained and made live for the bot responses to reach the Livechat widget.
+    * Refer to webex connect docs for a step-by-step guide on creating your first Q&A bot
+
+## Q&A bot flow for SMS channel
+Pre-requisites
+1.	A  Webex Connect tenant with bot builder app, WxCC nodes, and SMS channel enabled
+2.	A SMS number in Webex Connect
+3.  Q&A bot in the bot builder
+    * A Q&A bot with desired articles should be created in the bot builder app. This will create an imibot integration in Webex connect, which can then be selected from a       dropdown in the Q&A bot node.
+    * The Q&A bot should be trained and made live for the bot responses to reach the sms widget.
+    * Refer to webex connect docs for a step-by-step guide on creating your first Q&A bot
+
+## Task bot flow for Facebook channel
+Pre-requisites
+(Optional) This sample flow leverages a google sheet for fetching user info and adding user info to the said google sheet via sheet2api APIs. If you want to emulate this flow, you will need:
+
+1. A publicly accessible google sheet and a sheet2api account that has access to your google sheet. Refer to sheet2api documentation on how to make requests to         your sheet
```

Comment:
can we explain in last of doc if required what is sheet2api account.

Reply:
@siramana , @keshshar 
Do we have any doc explaining about sheet2api account

Reply:
i feel we should not get into that, we can give sheet2api documentation link https://sheet2api.com/documentation/

Reply:
Sure Thanks Siva